make_icons.py

```diff
@@ -22,7 +22,7 @@
 import unvtray
 from unvtray.tray import make_icon
 
-APP_NAME = unvtray.__name__  # pyright: ignore
+APP_NAME = "net.unvanquished." + unvtray.__name__  # pyright: ignore
```

Comment:
For consistency I'd prefer `f"net.unvanquished.{unvtray.__name__}"` or simply `"net.unvanquished.unvtray"` (then without the `pyright` hint).